### PR TITLE
Add authorizeByTag

### PR DESCRIPTION
**Pull request by Devgrapher**
This implements `authorizeByTag`, which is added at https://github.com/ridi/cms-sdk/pull/48.

### CODE REVIEW

File: src/Service/AdminTagService.php [hunk 1]
```diff
@@ -4,7 +4,7 @@
 use Ridibooks\Cms\Model\AdminTag;
 use Ridibooks\Cms\Thrift\AdminTag\AdminTag as ThriftAdminTag;
 use Ridibooks\Cms\Thrift\AdminTag\AdminTagServiceIf;
-use Ridibooks\Cms\Thrift\ThriftService;
+use Ridiboks\Cms\Thrift\ThriftService;
```
Comment (imkitae) [inline src/Service/AdminTagService.php]: Here is a typo

File: src/Service/AdminTagService.php [hunk 2]
```diff
@@ -60,4 +60,23 @@ public function getAdminTags(array $tag_ids): array
 
         return $tags;
     }
+
+    public function findTagByName(string $name): ?int
```
Comment (krazijames) [inline src/Service/AdminTagService.php]: 리턴값으로 보면 `findTagIdByName`이 맞지 않을까요?

File: src/Service/AdminTagService.php [hunk 3]
```diff
@@ -60,4 +60,23 @@ public function getAdminTags(array $tag_ids): array
...
+            return null;
+        }
+
+        return $tag_id;
+    }
+
+    public function findTagsByName(array $tag_names): array
+    {
+        foreach ($tag_names as $name) {
+            $tag_ids[] = self::findTagByName($name);
```
Comment (imkitae) [inline src/Service/AdminTagService.php]: `AdminTag::whereIn` seems better I think

File: src/Service/AdminTagService.php (hunk 2 above)
Comment (imkitae) [reply src/Service/AdminTagService.php]: I prefer to maintain code convention. I think it would be better to fix all or leave it aside.

**Pull request merged**